sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/implementation/StorageAccountImpl.java

```diff
@@ -240,6 +240,22 @@ public boolean isSharedKeyAccessAllowed() {
         return this.innerModel().allowSharedKeyAccess();
     }
 
+    @Override
+    public boolean isAllowCrossTenantReplication() {
+        if (this.innerModel().allowCrossTenantReplication() == null) {
+            return true;
```

Comment:
Any doc or evidence to support this behavior(if null, then true)?

Reply:
Yeah, please check Portal or CLI.

If they didn't show, it be safer just use `Boolean` and allow null.

Reply:
Now fixed to `return ResourceManagerUtils.toPrimitiveBoolean(this.innerModel().allowCrossTenantReplication());`.  The another one also had the same fix.

Reply:
Could you confirm whether default (when send `null`), it is true or false?